7a_temp_coop_munge/tmp/parser_files.yml

```diff
@@ -15,7 +15,7 @@
 7a_temp_coop_munge/src/data_parsers/parse_redlake_files.R: d6d3b66abb67ccad99ddeeb548fe867f
 7a_temp_coop_munge/src/data_parsers/parse_tenmile_files.R: d600cd9b651b20b4b47957341efae22b
 7a_temp_coop_munge/src/data_parsers/parse_test_coop_data.R: 1cac7a31dc074a4fd019133e4abbd276
-7a_temp_coop_munge/src/data_parsers/parse_UniversityofMissouri_LimnoProfiles_2017_2020.R: 4607f3052429313c2afeea56e631d611
+7a_temp_coop_munge/src/data_parsers/parse_UniversityofMissouri_LimnoProfiles_2017_2020.R: dac743a6cae59dcaf9c9bfedfe88cd95
```

Comment:
I don't know what could be driving that hash update - do you have any thoughts on how to run that down? It looks like it updated only after I re-ran the pipeline through `scamke('8_viz')` the [first time](https://github.com/USGS-R/lake-temperature-model-prep/pull/321/commits/594c44db9221b66a8cb16d3a8f616a48f61998bb)

Reply:
I would usually go and click the "history" tab on GitHub next to those files to see when they were last updated. The actual parser script was last updated by Jordan on March 11. 

![image](https://user-images.githubusercontent.com/13220910/160931446-40189610-617f-4000-a896-5f4b9b357ae8.png)

Looks like the update to this parser hash file was also updated when Jordan made the parser update

![image](https://user-images.githubusercontent.com/13220910/160931538-5dad4520-ae31-47d9-92c7-b1f10f799c94.png)

And looking at the hash change (the original was `4607f3052429313c2afeea56e631d611` and your change is to `dac743a6cae59dcaf9c9bfedfe88cd95`, it would make sense that you have made changes to that file but we aren't seeing any here. The original hash in your PR that is being changed, lines up with the last change committed by Jordan

![image](https://user-images.githubusercontent.com/13220910/160931813-1f3f7fcd-9585-4911-b301-6dcae6ae4efe.png)

Reply:
And you definitely don't have any uncommitted file changes? I wonder if there is a line-ending change? Jordan is on a Mac and I assume you are Windows, so sometimes there are some differences there. Do you have this box checked in your RStudio Git "diff" window?

![image](https://user-images.githubusercontent.com/13220910/160932270-9a382c9a-7ddb-431a-bddf-d20a74877994.png)

Reply:
I don't have that set to ignore whitespace - should I change that? Also, I opened two versions `parse_UniversityofMissouri_LimnoProfiles_2017_2020.R`: one from `main` and one from `fix-ia-xwalk-and-map`. Using a file compare, they both match.

I do recall Jordan's PR - it was to provide an example for me that used `case_when` to match parsers to files instead of filtering a vector of file names into different groups and then passing each group to each parser.

Reply:
OK, I would try checking "ignore whitespace" and see if anything pops up in your git commit window

Reply:
Checked "ignore whitespace" in rstudio git "diff" window and I don't see any file changes. Also, I typically use git via git bash, not rstudio - I'm not sure if this would make a difference or nto.

Reply:
I'll make a small task to follow up with jread next week.